internal/k8sinternal/client.go

```diff
@@ -207,5 +210,10 @@ func (kc kubeClient) GetKubernetesVersion() (*version.Info, error) {
 
 // ServerPreferredResources returns the supported resources with the version preferred by the server.
 func (kc kubeClient) ServerPreferredResources() ([]*metav1.APIResourceList, error) {
-	return discovery.ServerPreferredResources(kc.discoveryClient)
+	list, err := discovery.ServerPreferredResources(kc.discoveryClient)
+	// If a group is not served by the cluster the resources of this group will not be audited.
+	if _, ok := err.(*discovery.ErrGroupDiscoveryFailed); ok {
```

Comment:
Can we use `errors.Is`?

Reply:
> Can we use `errors.Is`?

`errors.Is` needs an error instance right? We want to check its type here.

Reply:
`errors.As` then? 😄